lldb/tools/lldb-dap/JSONUtils.cpp

```diff
@@ -416,8 +416,11 @@ llvm::json::Value CreateModule(lldb::SBTarget &target, lldb::SBModule &module,
   } else {
     object.try_emplace("symbolStatus", "Symbols not found.");
   }
-  std::string loaded_addr = std::to_string(
-      module.GetObjectFileHeaderAddress().GetLoadAddress(target));
+  std::string loaded_addr;
+  llvm::raw_string_ostream os_hex(loaded_addr);
+  os_hex << llvm::format_hex(
+      module.GetObjectFileHeaderAddress().GetLoadAddress(target),
+      sizeof(lldb::addr_t));
```

Comment:
I assumed the load address was a "number" but the [spec](https://microsoft.github.io/debug-adapter-protocol/specification#Types_Module) says it's a "string" in which case I agree it makes more sense to the hex conversion here. 

You can  simplify this something like this:
```
object.try_emplace("addressRange", llvm::format({0:x}, module.GetObjectFileHeaderAddress().GetLoadAddress(target)).str());
```

Reply:
https://github.com/microsoft/debug-adapter-protocol/issues/424 was filed to clarify how the address range is supposed to be represented. It hasn't had any movement recently, so we could ping the issue to see if the folks that work on the DAP have any more recent thoughts.